2-mandatory/1-writers.js

```diff
@@ -60,9 +60,13 @@ Exercise 1:
   "Hi, my name is {firstName} {lastName}. I am {age} years old, and work as a {occupation}."
 */
 function logAllWriters() {
-  // write your code to log all writers here
+  writers.forEach(function(e){
+    let str = `Hi, my name is ${e.firstName} ${e.lastName}. I am ${e.age} years old, and work as a ${e.occupation}`
+    console.log(str)
```

Comment:
This is correct, you could also directly console.log the string without setting it as a variable first